environment-dev.yml

```diff
@@ -31,6 +31,7 @@ dependencies:
   - sqlalchemy>=2.0.0
   - pandas-gbq>=0.19.0
   - pytables>=3.8.0
+  - c-blosc2<=2.14.1
```

Comment:
Should we also update requirements-dev.txt?

Reply:
I think this is not necessary, we just need the tests in CI to work as quickly as possible, I believe that either `pytables` or `c-blosc2` will release their new patches to make everything work. Also, I'm not sure if this error is present when installing using pip.